src/main/java/org/sonarlint/intellij/actions/OpenInBrowserAction.kt

```diff
@@ -19,12 +19,12 @@
  */
 package org.sonarlint.intellij.actions
 
-import com.intellij.icons.AllIcons
 import com.intellij.ide.BrowserUtil
 import com.intellij.openapi.actionSystem.AnActionEvent
+import javax.swing.Icon
 
-class OpenInBrowserAction(text: String, description: String?, private val link: String) : AbstractSonarAction(
-    text, description, AllIcons.Actions.Help
+class OpenInBrowserAction(text: String, description: String?, private val link: String, icon: Icon?) : AbstractSonarAction(
```

Comment:
You could put a default value `, icon: Icon? = null` so you don't have to add `null` everywhere